WindDirectionControl/WindDirectionControl.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Agate styled WindDirectionControl component and behavior.
+ *
+ * @example
+ * <WindDirectionControl />
+ *
+ * @module agate/WindDirectionControl
+ * @exports WindDirectionControl
+ * @exports WindDirectionControlBase
+ * @exports WindDirectionControlDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Changeable from '@enact/ui/Changeable';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import ArcPicker from '../ArcPicker';
+import Icon from '../Icon';
+import Skinnable from '../Skinnable';
+
+import css from './WindDirectionControl.module.less';
+
+/**
+ * An Agate component for displaying Wind Direction Control {@link agate/WindDirectionControl}.
+ *
+ * @class WindDirectionControl
```

Comment:
Let's define `WindDirectionControlBase` here and `WindDirectionControl` at 113 line - https://github.com/enactjs/agate/pull/339/files#diff-0cfb1ecd7dd823fe639f727f23666809R113 line.
```suggestion
 * @class WindDirectionControlBase
```